{"timestamp": "2025-08-28T00:32:40.716431", "level": "INFO", "message": "\u001b[2m2025-08-27T16:32:40.716221Z\u001b[0m [\u001b[32m\u001b[1minfo     \u001b[0m] \u001b[1mlogging_initialized           \u001b[0m [\u001b[0m\u001b[1m\u001b[34mapp.core.logging\u001b[0m]\u001b[0m \u001b[36menvironment\u001b[0m=\u001b[35mdevelopment\u001b[0m \u001b[36mfilename\u001b[0m=\u001b[35mlogging.py\u001b[0m \u001b[36mfunc_name\u001b[0m=\u001b[35m<module>\u001b[0m \u001b[36mlineno\u001b[0m=\u001b[35m177\u001b[0m \u001b[36mlog_format\u001b[0m=\u001b[35mconsole\u001b[0m \u001b[36mlog_level\u001b[0m=\u001b[35mDEBUG\u001b[0m \u001b[36mmodule\u001b[0m=\u001b[35mlogging\u001b[0m \u001b[36mpathname\u001b[0m=\u001b[35mC:\\Workspace\\AI\\Project\\AgentsFramework\\app\\core\\logging.py\u001b[0m", "module": "logging", "function": "<module>", "filename": "C:\\Workspace\\AI\\Project\\AgentsFramework\\app\\core\\logging.py", "line": 177, "environment": "development"}
{"timestamp": "2025-08-28T00:32:47.793236", "level": "ERROR", "message": "\u001b[2m2025-08-27T16:32:47.793020Z\u001b[0m [\u001b[31m\u001b[1merror    \u001b[0m] \u001b[1mdatabase_initialization_error \u001b[0m [\u001b[0m\u001b[1m\u001b[34mapp.core.logging\u001b[0m]\u001b[0m \u001b[36menvironment\u001b[0m=\u001b[35mdevelopment\u001b[0m \u001b[36merror\u001b[0m=\u001b[35m'(psycopg2.OperationalError) connection to server at \"localhost\" (::1), port 5432 failed: Connection refused (0x0000274D/10061)\\n\\tIs the server running on that host and accepting TCP/IP connections?\\nconnection to server at \"localhost\" (127.0.0.1), port 5432 failed: Connection refused (0x0000274D/10061)\\n\\tIs the server running on that host and accepting TCP/IP connections?\\n\\n(Background on this error at: https://sqlalche.me/e/20/e3q8)'\u001b[0m \u001b[36mfilename\u001b[0m=\u001b[35mdatabase.py\u001b[0m \u001b[36mfunc_name\u001b[0m=\u001b[35m__init__\u001b[0m \u001b[36mlineno\u001b[0m=\u001b[35m62\u001b[0m \u001b[36mmodule\u001b[0m=\u001b[35mdatabase\u001b[0m \u001b[36mpathname\u001b[0m=\u001b[35mC:\\Workspace\\AI\\Project\\AgentsFramework\\app\\services\\database.py\u001b[0m", "module": "database", "function": "__init__", "filename": "C:\\Workspace\\AI\\Project\\AgentsFramework\\app\\services\\database.py", "line": 62, "environment": "development"}
{"timestamp": "2025-08-28T00:33:26.387192", "level": "INFO", "message": "\u001b[2m2025-08-27T16:33:26.387010Z\u001b[0m [\u001b[32m\u001b[1minfo     \u001b[0m] \u001b[1mlogging_initialized           \u001b[0m [\u001b[0m\u001b[1m\u001b[34mapp.core.logging\u001b[0m]\u001b[0m \u001b[36menvironment\u001b[0m=\u001b[35mdevelopment\u001b[0m \u001b[36mfilename\u001b[0m=\u001b[35mlogging.py\u001b[0m \u001b[36mfunc_name\u001b[0m=\u001b[35m<module>\u001b[0m \u001b[36mlineno\u001b[0m=\u001b[35m177\u001b[0m \u001b[36mlog_format\u001b[0m=\u001b[35mconsole\u001b[0m \u001b[36mlog_level\u001b[0m=\u001b[35mDEBUG\u001b[0m \u001b[36mmodule\u001b[0m=\u001b[35mlogging\u001b[0m \u001b[36mpathname\u001b[0m=\u001b[35mC:\\Workspace\\AI\\Project\\AgentsFramework\\app\\core\\logging.py\u001b[0m", "module": "logging", "function": "<module>", "filename": "C:\\Workspace\\AI\\Project\\AgentsFramework\\app\\core\\logging.py", "line": 177, "environment": "development"}
{"timestamp": "2025-08-28T00:33:33.236774", "level": "ERROR", "message": "\u001b[2m2025-08-27T16:33:33.236567Z\u001b[0m [\u001b[31m\u001b[1merror    \u001b[0m] \u001b[1mdatabase_initialization_error \u001b[0m [\u001b[0m\u001b[1m\u001b[34mapp.core.logging\u001b[0m]\u001b[0m \u001b[36menvironment\u001b[0m=\u001b[35mdevelopment\u001b[0m \u001b[36merror\u001b[0m=\u001b[35m'(psycopg2.OperationalError) connection to server at \"localhost\" (::1), port 5432 failed: Connection refused (0x0000274D/10061)\\n\\tIs the server running on that host and accepting TCP/IP connections?\\nconnection to server at \"localhost\" (127.0.0.1), port 5432 failed: Connection refused (0x0000274D/10061)\\n\\tIs the server running on that host and accepting TCP/IP connections?\\n\\n(Background on this error at: https://sqlalche.me/e/20/e3q8)'\u001b[0m \u001b[36mfilename\u001b[0m=\u001b[35mdatabase.py\u001b[0m \u001b[36mfunc_name\u001b[0m=\u001b[35m__init__\u001b[0m \u001b[36mlineno\u001b[0m=\u001b[35m62\u001b[0m \u001b[36mmodule\u001b[0m=\u001b[35mdatabase\u001b[0m \u001b[36mpathname\u001b[0m=\u001b[35mC:\\Workspace\\AI\\Project\\AgentsFramework\\app\\services\\database.py\u001b[0m", "module": "database", "function": "__init__", "filename": "C:\\Workspace\\AI\\Project\\AgentsFramework\\app\\services\\database.py", "line": 62, "environment": "development"}
